pkg/common/fileutils/fileutils_test.go

```diff
@@ -50,3 +50,59 @@ func Test_GetAllFilesWithSameNameInDirectory(t *testing.T) {
 	assert.Equal(t, 4, len(retVal))
 	assert.ElementsMatch(t, expectedPaths, retVal)
 }
+
+func Test_GetAllFilesWithSpecificExtensions(t *testing.T) {
+	temp := t.TempDir()
+	_, err := os.Create(temp + "/abc.xyz")
+	assert.Nil(t, err)
+	err = os.Mkdir(temp+"/dir1", 0755)
+	assert.Nil(t, err)
+	err = os.Mkdir(temp+"/dir2", 0755)
+	assert.Nil(t, err)
+	err = os.Mkdir(temp+"/dir1/dir3", 0755)
+	assert.Nil(t, err)
+	err = os.Mkdir(temp+"/dir2/dir4", 0755)
+	assert.Nil(t, err)
+	_, err = os.Create(temp + "/dir1/abc.txt")
+	assert.Nil(t, err)
+	_, err = os.Create(temp + "/dir1/abc.mno")
+	assert.Nil(t, err)
```

Comment:
nit: this is getting repetitive (along with your above function `Test_GetAllFilesWithSameNameInDirectory`). You can keep it like this if you want, but you might want to make a small helper function like `forceCreate(filename)` that makes the dir and creates the file and fails the test if either of those gave an error